R/description.R

```diff
@@ -1,13 +1,20 @@
 #' Create or modify a DESCRIPTION file
 #'
 #' @description
+#'
+#' `use_description()` creates a `DESCRIPTION` file. `DESCRIPTION` files are
+#' also useful in non-package projects, particularly for declaring dependencies,
+#' as a project `DESCRIPTION` allows users to install required packages with
+#' [`devtools::install_deps()`]. Note that `use_decription()` checks that the
+#' project has a valid package name, which isn't necessary for a project; turn
+#' this off with `check_name = FALSE`.
+#'
 #' usethis consults the following sources, in this order, to set `DESCRIPTION`
-#' fields:
-#' * `fields` argument of [create_package()] or [use_description()].
-#' * `getOption("usethis.description")` or `getOption("devtools.desc")`. The
+#' fields: * `fields` argument of [create_package()] or [use_description()]. *
+#' `getOption("usethis.description")` or `getOption("devtools.desc")`. The
 #' devtools option is consulted only for backwards compatibility and it's
-#' recommended to switch to an option named "usethis.description".
-#' * Defaults built into usethis.
+#' recommended to switch to an option named "usethis.description". * Defaults
+#' built into usethis.
```

Comment:
This looks like a mangled bullet list. I suspect it was partially mangled to start with. This happens when I reflow roxygen comments. Can you straighten it out?

Reply:
ack, will do!